doc/programming_guide/sound.rst

```diff
@@ -0,0 +1,724 @@
+playback of the movement sound.
+
+This is crucial for games which hide parts of the world from view.
+An enemy with no way for the user to know it's there is the most
+common version of the unknown danger mentioned in :ref:`sound-why-important`.
+
+See the following to learn more:
+
+#. :ref:`Platformer Tutorial - Part 7 - Collision Detection <platformer_part_seven_playing_sounds>`
+#. :ref:`sound_demo`
+
+.. _sound-basics-stopping:
+
+Stopping Sounds
+^^^^^^^^^^^^^^^
+
+
+Arcade's helper functions are the easiest way to stop playback. To use them, do
+the following:
+
+#. Choose one of the following:
+
+   * Pass the stored pyglet :py:class:`~pyglet.media.player.Player` to
+     :py:func:`arcade.stop_sound`::
+
+      arcade.stop_sound(self.current_playback)
+
+   * Pass the stored pyglet :py:class:`~pyglet.media.player.Player` to the
+     sound's :py:meth:`~arcade.Sound.stop` method::
+
+      self.hurt_sound.stop(self.current_playback)
+
+
+#. Clear any references to the player to allow its memory to be freed::
+
+    # For each object, Python tracks how many other objects use it. If
+    # nothing else uses an object, it will be marked as garbage which
+    # Python can delete automatically to free memory.
+    self.current_playback = None
+
+See the following to learn more:
+
+* :ref:`sound-compat-easy`
+* :ref:`sound-advanced-playback`
+
+.. _sound-loading-modes:
+
+Streaming or Static Loading?
+----------------------------
+
+.. _keyword argument: https://docs.python.org/3/glossary.html#term-argument
+
+.. list-table::
+   :header-rows: 1
+
+   * - Streaming
+     - Best [#meaningbestformatheader]_ Format
+     - Decompressed
+     - Best Uses
+
+   * - ``False`` (Default)
+     - ``.wav``
+     - Whole file
+     - 2+ overlapping playbacks, short, repeated, unpredictable
+
+   * - ``True``
+     - ``.mp3``
+     - Predicted data
+     - 1 copy & file at a time, long, uninterrupted
+
+By default, arcade decompresses the entirety of each sound into memory.
+
+This is the best option for most game sound effects. It's called
+"static" [#staticsourcefoot]_ audio because the data never changes.
+
+The alternative is streaming. Enable it by passing ``True`` through the
+``streaming`` `keyword argument`_  when you :ref:`load a sound
+<sound-basics-loading>`::
+
+    # Both loading approaches accept the streaming keyword.
+    classical_music_track = arcade.load_sound(":resources:music/1918.mp3", streaming=True)
+    funky_music_track = arcade.Sound(":resources:music/funkyrobot.mp3", streaming=True)
+
+
+For an interactive example, see the :ref:`music_control_demo`.
+
+The following subheadings will explain each option in detail.
+
+.. [#meaningbestformatheader]
+   See :ref:`sound-compat-easy` to learn more.
+
+.. [#staticsourcefoot]
+   See the :py:class:`pyglet.media.StaticSource` class used by arcade.
+
+.. _sound-loading-modes-static:
+
+Static Sounds Can Be Fastest
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Static sounds can be faster than streaming sounds

Reply:
I would like to use a longer title, but this will start getting ugly in the Furo sidebar. Were' already at the line length limit.

Current:
![image](https://github.com/pythonarcade/arcade/assets/36696816/07c878af-cbdb-4637-a820-a3319d9c2517)
Proposed:
![image](https://github.com/pythonarcade/arcade/assets/36696816/cd0a6737-a44f-4e83-9440-aca72c46d0e9)

Reply:
Marking this item as resolved since it got a thumbs up on Discord.